lib/serverpipe.js

```diff
@@ -58,11 +58,25 @@ export function postMultipartFormWithCredentials(url, form, options = {}) {
     Accept: 'application/json',
   };
   const headers = 'headers' in options ? [...options.headers, defaultHeaders] : defaultHeaders;
-
   headers[getCsrfHeaderName()] = getCsrfToken();
+
+  const fileInputs = form.querySelectorAll('input[type="file"]:not([disabled])')
```

Comment:
Could you add a tiny comment to reference the original JIRA, as without any context it looks like abject madness